core/src/contact_info.rs

```diff
@@ -10,6 +10,9 @@ use std::net::{IpAddr, SocketAddr};
 /// Structure representing a node on the network
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct ContactInfo {
+    /// validator identity
```

Comment:
this is going to break gossip, we'll need to figure out how to roll this out to mainnet-beta 🤔